sys-libs/basu/files/0002-basu-0.2.0-meson-fix-linking-on-LLD-13-LD-2.37-and-older-CCs.patch

```diff
@@ -0,0 +1,41 @@
+From 80362bad0d61687c610a8790679188e8e0e9ce2d Mon Sep 17 00:00:00 2001
```

Comment:
Link to any relevant upstream links + gentoo bugs at the top of each patch

Reply:
Done